vendor/update-opam-file-format.sh

```diff
@@ -1,20 +1,26 @@
 #!/bin/bash
 
-version=2.1.5
+version=35d311bd38d1d83414b7095b702ae24d99885a99
+name=opam-file-format
 
 set -e -o pipefail
 
 TMP="$(mktemp -d)"
 trap "rm -rf $TMP" EXIT
 
-rm -rf opam-file-format
-mkdir -p opam-file-format/src
+rm -rf $name
+mkdir -p $name/src
 
-(cd $TMP && opam source opam-file-format.$version)
+(
+    cd $TMP
+    git clone https://github.com/ocaml-dune/$name.git
+    cd $name
+    git checkout $version
+)
 
-SRC=$TMP/opam-file-format.$version
+SRC=$TMP/$name
 
-cp -v $SRC/src/*.{ml,mli,mll,mly} opam-file-format/src
+cp -v $SRC/src/*.{ml,mli,mll,mly} $name/src
+cp -v $SRC/src/dune $name/src
 
-git checkout opam-file-format/src/dune
```

Comment:
how come this line was removed?

Reply:
Now the `dune` file changes are [vendored](https://github.com/ocaml-dune/opam-file-format/commit/35d311bd38d1d83414b7095b702ae24d99885a99), I thought that is a better place to track the patch stack than in dune itself, as we have to keep a fork anyway.

Reply:
The changes to the dune files themselves is something we keep in the dune repo actually. In fact, we don't even track the upstream dune files and write them all ourselves. So there's no need to patch them.

Note that in this PR, we have a slightly different build in bootstrap (which doesn't use the dune files at all) and the normal build we get from `./dune build`. So it would be good to just write a minimal dune file to remove the differences.

Reply:
Ok, I've minimized the changes vs the dune file that we have. Wasn't very much in the end.